scripts/update/Updater.php

```diff
@@ -2124,7 +2124,7 @@ public function update($initialVersion)
             $this->setVersion('38.13.0');
         }
 
-      $this->skip('38.13.0', '38.14.0');
+      $this->skip('38.13.0', '38.16.1');
```

Comment:
i think now it is prohibited to update this file

Reply:
done